build.gradle

```diff
@@ -114,6 +115,9 @@ subprojects {
 
         testRuntime("javax.el:javax.el-api")
         testRuntime("org.glassfish:javax.el")
+
+        testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
```

Comment:
You don't need to include these versions here as it is already included in the spring platform BOM we're using. We default to that as much as possible that way when we upgrade the BOM version the entire system upgrades compatible libraries. See [here](http://docs.spring.io/platform/docs/2.0.6.RELEASE/reference/htmlsingle/).